src/main/scala/chisel3/stage/phases/Convert.scala

```diff
@@ -29,8 +29,7 @@ class Convert extends Phase {
       /* Convert all Chisel Annotations to FIRRTL Annotations */
       a
         .circuit
-        .annotations
-        .map(_.toFirrtl) ++
+        .firrtlAnnotations ++
```

Comment:
I _thought_ there was another place we needed to change this, but I couldn't find it, so maybe I was wrong?

Reply:
Yeah I couldn't find it either, if it comes up it's a bug to fix!